### ISSUE TITLE
[Accessibility] Github.dev: in Get Started, headings can be expanded but this isn't conveyed to a screen reader

### ISSUE THREAD

**Issue by devinprater** [microsoft/vscode#135093]
Does this issue occur when all extensions are disabled?: Yes/No
N/a

- VS Code Version: Server-side
- OS Version: Windows 11: Version 21H2 (OS Build 22000.194)
on Microsoft Edge

Steps to Reproduce:

1. Sign into Github and open a repository.
2. In the repository, press the period (.) key.
3. Navigate through the getting started area. Headings read like 

> Limitless extensibility
>  clickable"

4. Press Enter on one of the headings.
5. New information will be revealed.

@isidorn

**Comment by isidorn** [comment]
I have investigated the HTML structure and I see you nicely set `aria-expanded` - great!
However you also use the default `listitem` role. I suggest to instead use role `button`.
Looking at the docs `aria-expanded` is not supported with `listitem` and it works with the `button` which should be the right role here.
You can override the list element role by using the ListAccessibilityProvider. [Example](https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/debug/browser/breakpointsView.ts#L965)

**Comment by JacksonKearl** [comment]
Thanks for the good analysis!

**Comment by rchiodo** [comment]
/verified

Narrator read the sections and developer tools showed the new role.
